src/main/java/dan200/computercraft/core/lua/LuaJLuaMachine.java

```diff
@@ -412,6 +415,7 @@ public void execute()
                                             }
                                             catch( Throwable t )
                                             {
+                                                ComputerCraft.log.error( "Error running task", t);
```

Comment:
While no well behaved peripheral should ever throw an exception other than LuaException, some so, so this introduces another mechanism for a malicious flood the server log.  Seeing as the error is returned to the lua user anyway, could we remove this line?